packages/resource-deployment/scripts/pool-startup/pool-startup-internal.sh

```diff
@@ -15,6 +15,8 @@ waitForAptUpdates() {
     done
 }
 
+waitForAptUpdates
```

Comment:
since the function is used to wait for locks other than "apt-get update", should we change the function name?